docker-compose-develop.yml

```diff
@@ -8,7 +8,10 @@ services:
     environment:
       PORT: 4400
       NODE_PATH: app/src
-      CT_URL: https://api.resourcewatch.org
+      NODE_ENV: dev
+      LOGGER_LEVEL: debug
+      SUPPRESS_NO_CONFIG_WARNING: "true"
+      CONTROL_TOWER_URL: https://api.resourcewatch.org
```

Comment:
Isn't control tower a thing of the past? Should these be renamed? Maybe in another PR perhaps?